config/bundles.php

```diff
@@ -20,4 +20,6 @@
     SymfonyCasts\Bundle\VerifyEmail\SymfonyCastsVerifyEmailBundle::class => ['all' => true],
     KnpU\OAuth2ClientBundle\KnpUOAuth2ClientBundle::class => ['all' => true],
     Scheb\TwoFactorBundle\SchebTwoFactorBundle::class => ['all' => true],
+    SpomkyLabs\CborBundle\SpomkyLabsCborBundle::class => ['all' => true],
+    Webauthn\Bundle\WebauthnBundle::class => ['all' => true],
```

Comment:
i see that you need on twig files `templates/user/login.html.twig` & `templates/user/manage_authenticators.html.twig` some javascript, than seems to be kind of generic no?
maybe the bundle can provide them?
a raw js file (or perhaps a stimulus controller)